api/build.gradle

```diff
@@ -440,7 +441,8 @@ dependencies {
   }
 
   compile "org.springframework.retry:spring-retry:1.1.5.RELEASE"
-  compile "org.springframework.security:spring-security-web:4.2.15.RELEASE"
```

Comment:
Updating only "web" did not resolve this as expected.  A still-vulnerable older "core" was still selected for some reason.  Explicitly setting both solved the problem.